Source/Fuse.Controls.ScrollView/Scroller.uno

```diff
@@ -276,6 +296,8 @@ namespace Fuse.Gestures
 
 		GestureRequest IGesture.OnPointerReleased(PointerReleasedArgs args)
 		{
+			StopInvalidateVisual();
```

Comment:
This is not necessary, the one in `OnLostCapture` is enough (it is always called).